code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/PopkovKirillTest.java

```diff
@@ -0,0 +1,65 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class PopkovKirillTest {
+    private final double delta = 0.001;
+
+    @Test
+    public void initComplexNumberWithEmptyConstructor() {
+        ComplexNumber cn = new ComplexNumber();
+
+        assertEquals("0.0 + 0.0i", cn.toString());
+    }
+
+    @Test
+    public void getRealPartOfComplexNumber() {
+        ComplexNumber cn = new ComplexNumber(1, 2);
+
+        assertEquals(1.0, cn.getReal(), delta);
+    }
+
+    @Test
+    public void getImaginaryPartOfComplexNumber() {
+        ComplexNumber cn = new ComplexNumber(1, 2);
+
+        assertEquals(2.0, cn.getImaginary(), delta);
+    }
+
+    @Test
+    public void getRealPartOfInitWithStringsComplexNumber() {
+        ComplexNumber cn = new ComplexNumber("1", "2");
+
+        assertEquals(1.0, cn.getReal(), delta);
+    }
+
+    @Test
+    public void getImaginaryPartOfInitWithStringsComplexNumber() {
+        ComplexNumber cn = new ComplexNumber("1", "2");
+
+        assertEquals(2.0, cn.getImaginary(), delta);
+    }
+
+    @Test
+    public void checkAdditionOfComplexNumbers() {
+        ComplexNumber cn1 = new ComplexNumber("1", "2");
+        ComplexNumber cn2 = new ComplexNumber("2", "3");
+        ComplexNumber cn3 = cn1.add(cn2);
+
+        assertEquals(3.0, cn3.getReal(), delta);
+        assertEquals(5.0, cn3.getImaginary(), delta);
```

Comment:
Why do you need two Asserts instead of one? According AAA it should be only one Assert.
It is easier to replace it with 

assertEquals(new ComplexNumber(3, 5), cn3, delta);

And in other tests too.